app/controllers/physical_server_controller.rb

```diff
@@ -33,7 +33,7 @@ def show_list
   end
 
   def textual_group_list
-    []
+    [%i(properties relationships assets)]
```

Comment:
I think this should come as a follow up to https://github.com/ManageIQ/manageiq-ui-classic/pull/1294